.circleci/scripts/deps-install.sh

```diff
@@ -8,4 +8,3 @@ set -o pipefail
 yarn --frozen-lockfile --ignore-scripts --har
 
 # install script whitelist
```

Comment:
I'm guessing we also don't need this comment?

Reply:
Removed!